fuse/core.go

```diff
@@ -51,6 +52,14 @@ func getIDs() (uint32, uint32) {
 
 var uid, gid = getIDs()
 
+func joinPath(parentPath string, e plugin.Entry) string {
```

Comment:
Sorry to re-introduce this again. Figured it's better long-term, since it separates the FUSE concept of a path (which is filesystemy) from Wash's concept (which right now is filesystemy, but may not be in the future depending on how annoying the issue described in https://github.com/puppetlabs/wash/issues/156 is)